### PR TITLE
list references for contributing to open source closes #86

### LINKED ISSUE #86: add resources on how to start contributing to open source
we need some more good articles on this topic.
Please suggest them here.

### PR DESCRIPTION
**Pull request by DomBert**
I started collecting some blogs and anchors for beginning to contribute in open source. It's a first draft I want to push.

### CODE REVIEW

File: biblio.Rmd [hunk 1]
```diff
@@ -115,3 +115,39 @@ pandoc homework.md --bibliography=mybib.bib -o .\Outputs\output.html
...
 where `--bibliography=mybib.bib` specifies that you want to use your `mybib.bib`, expected at the project root.
 This would be the file that you'd have exported from Zotero.
+
+## Resources on How to Start Contributing to Open Source
+
+Mastering the entry into open source can be frightening. To make your contribution easier, a collection of links has been compiled here. First of all, it makes sense to spend a bit of time on blogs on this topic.
+
+### Blogs and Anchors
+
+A **First Timers Only: For absolute Newbies**
```
Comment (maxheld83) [inline biblio.Rmd]: you can use `#### Fourth Level Headings` for this.

**Comment by maxheld83** [comment]
@DomBert thanks! These Appetizer quotes are really great!

File: biblio.Rmd (hunk 1 above)
Comment (DomBert) [reply biblio.Rmd]: **done**

File: biblio.Rmd [hunk 2]
```diff
@@ -115,3 +115,39 @@ pandoc homework.md --bibliography=mybib.bib -o .\Outputs\output.html
...
+
+A nice introduction to the topic is the initiative of *SHANSELMAN* and *KENTCDODDS*. 
+
+Start [here](https://github.com/shanselman/firsttimersonly). 
+**Appetizer** *We believe - and we hope you do too - that learning how to code, how to think, and how to contribute to open source can empower the next generation of coders and creators. We VALUE first time contributors and we want them to know that everyone started somewhere! We have gathered a list of resources to motivate and empower you to making that first step into becoming a contributor to the open source realm. If you are an OSS project owner, we have some tips for you to enable you to be instrumental in ushering a new batch of programmers into the community.*
```
[the remark this answers is not in the document]
Comment (DomBert) [reply biblio.Rmd]: Thanks for your feedback! I changed a lot and completed the site. 
Let me know, if I can improve more stuff!

**Pull request merged**